src/test/kotlin/io/libp2p/security/SecureChannelTest.kt

```diff
@@ -57,25 +70,36 @@ abstract class SecureChannelTest(
         protocolSelect2.selectedFuture.get(10, TimeUnit.SECONDS)
         logger.debug("Secured!")
 
-        val handler1 = SecureChannelTestHandler("1", latch)
-        val handler2 = SecureChannelTestHandler("2", latch)
+        val data1: String
+        val data2: String
+        if (dataSize > 2) {
+            val trimDataSize = dataSize - 2
+            val data = (0 until trimDataSize / 10)
+                .map { " x" + Integer.toHexString(it * 10).padStart(8, '0') }
+                .joinToString("")
+                .padEnd(trimDataSize, '_')
+            data1 = "1-$data"
+            data2 = "2-$data"
+        } else {
+            data1 = if (dataSize == 1) "1" else ""
+            data2 = if (dataSize == 1) "2" else ""
```

Comment:
Doesn't look like `dataSize` is ever `== 1`

Reply:
Fix the condition above. Yep there is no cases with size 1, but just made this piece of code universal :)